vector/src/main/java/im/vector/app/features/onboarding/ftueauth/MatrixOrgMissingRegistrationStagesComparator.kt

```diff
@@ -18,10 +18,10 @@ package im.vector.app.features.onboarding.ftueauth
 
 import org.matrix.android.sdk.api.auth.registration.Stage
 
-class FtueMissingRegistrationStagesComparator : Comparator<Stage> {
+class MatrixOrgMissingRegistrationStagesComparator : Comparator<Stage> {
```

Comment:
Nit: thinking again about the name of this class, I think the word `Missing` could be removed.